components/shopify/shopify.app.js

```diff
@@ -1,4 +1,8 @@
 const axios = require("axios");
+const Bottleneck = require('bottleneck');
+const limiter = new Bottleneck({
```

Comment:
I might just add a small comment here linking to the Spotify API docs on their rate limits, noting that the `minTime` was set as a result of their rate limit